app/components/StampSelector.tsx

```diff
@@ -71,14 +74,10 @@ export function StampSelector({
                       key={`${provider.title}${i}`}
                       data-testid={`indicator-${provider.name}`}
                     >
-                      <div className={`text-md relative top-[-0.3em] ${textColor}`}>
-                        {process.env.NEXT_PUBLIC_FF_CHAIN_SYNC === "on" &&
-                          (isProviderOnChain(provider.name) ? (
-                            <LinkIcon className="mr-2 inline h-6 w-5 text-accent-3" />
-                          ) : (
-                            <LinkIcon className="mr-2 inline h-6 w-5 text-color-4" />
-                          ))}
+                      <div className={`text-md relative top-[-0.3em] ${textColor} flex items-center`}>
                         {provider.title}
+                        {process.env.NEXT_PUBLIC_FF_CHAIN_SYNC === "on" &&
+                          (isProviderOnChain(provider.name) ? <OnChainTag marginLeft="3" /> : <></>)}
```

Comment:
You could remove the `<></>` with:
```suggestion
                          (isProviderOnChain(provider.name) && <OnChainTag marginLeft="3" />)}
```
What you have works too!